src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/EndpointContracts/IVSFindAllReferencesEndpoint.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.CommonLanguageServerProtocol.Framework;
+using Microsoft.VisualStudio.LanguageServer.Protocol;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.EndpointContracts
```

Comment:
file-scope namespace, on all these new files.

Reply:
Actually I'm surprised you didn't get some kind of warning/error about these, we should definitely make that louder.

Reply:
Yea I agree. I don't think I have full solution analysis on so that might have been why

Reply:
Should fail ci build and its not...